### ISSUE TITLE
[FTR] User online status in DM support

### ISSUE THREAD

**Issue by Martin5001** [austinhuang0131/barinsta#1065]
I understand this app is privacy oriented and some people don't want to make their online status visible, but I think the option should still be there for those who use this feature. (If that's even possible to implement for 3rd party app, of course.)

**Comment by ammargitham** [comment]
This is not possible until mqtt is implemented.

**Comment by austinhuang0131** [comment]
actully this is possible on REST, we just can't make it instant

**Comment by Androz2091** [comment]
Yes, this feature would be nice and useful! So it means that it does not show me online when using the app for the moment? As it does not use Instagram mqtt
